airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/AirbyteMessageTracker.java

```diff
@@ -238,12 +238,8 @@ private void handleEmittedOrchestratorMessage(final AirbyteControlMessage contro
   @SuppressWarnings("PMD") // until method is implemented
   private void handleEmittedOrchestratorConnectorConfig(final AirbyteControlConnectorConfigMessage configMessage,
                                                         final ConnectorType connectorType) {
-    // TODO: Update config here
-    /**
-     * Pseudocode: for (key in configMessage.getConfig()) { validateIsReallyConfig(key);
-     * persistConfigChange(connectorType, key, configMessage.getConfig().get(key)); // nuance here for
-     * secret storage or not. May need to be async over API for replication orchestrator }
-     */
+    // Config updates are being persisted as part of the DefaultNormalizationWorker.
+    // In the future, we could add tracking of these kinds of messages here. Nothing to do for now.
```

Comment:
That's a great idea! Story added for this: https://github.com/airbytehq/airbyte/issues/19951